src/client/components/share.stories.ts

```diff
@@ -16,16 +17,23 @@ interface ShareArgs {
     prompt?: string,
 }
 
-const Template = (args: ShareArgs) => html`
-    <dwac-share .target="${args.target ? new URL(args.target) : undefined}"
-            .articleTitle="${args.articleTitle}"
-            .prompt="${args.prompt}">
-    </dwac-share>
-`;
-
-export const Primary = Template.bind({});
-(Primary as any).args = {
-    target: 'https://blog.dwac.dev/',
-    articleTitle: 'Devel without a Cause',
-    prompt: 'Check out my blog!',
+export const primary = ({
+    target = 'https://blog.dwac.dev/',
+    articleTitle = 'Devel without a Cause',
+    prompt = 'Check out my blog!',
+}: ShareArgs = {}) => {
+    
+    return html`
+        <dwac-share .target="${target ? new URL(target) : undefined}"
+                article-title="${articleTitle}"
+                prompt="${prompt}">
+        </dwac-share>        
+    `
```

Comment:
This is a collision of v5 vs v6 CFS syntax. `@open-wc/demoing-storybook` only supports v5 at this time. There is research going into https://github.com/modernweb-dev/storybook-prebuilt where we might move to that in the future, but it's not currently "stable" for release just yet.

Reply:
Gotcha, so I want to use a function with default values for now rather setting an arguments object. Storybook must be doing some real magic to display these default values then, I can see why they wanted to move to explicitly exported args.